worldengine/World.proto

```diff
@@ -1,3 +1,4 @@
+syntax = "proto2";
```

Comment:
Yes, I've always commented/deleted that one line. It is automatically produced, but never consumed and pyflakes chokes on it.